### PR TITLE
Vs-eng-spanner-interactable-lv-522

### PR DESCRIPTION
**Pull request by RyanASwanson**
https://bradleycapstone.atlassian.net/jira/software/projects/LV/boards/32/backlog?assignee=6318bb796856bdd60aa045de

There is a spanner. It can be interacted with. No way

### CODE REVIEW

**Review by astapay — APPROVED** [review]
hooray!

**Review by AlecatEnjoyer — APPROVED** [review]
I approve

File: Assets/GameAssets/Player/PlayerScripts/PlayerInventory.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+/// Done here in order to have a centralized place for any pickups
+/// </summary>
+public class PlayerInventory : MonoBehaviour
+{
+    public static PlayerInventory Instance;
+
+    public bool DoesPlayerHaveSpanner { get; set; }
+
+    // Start is called before the first frame update
+    void Start()
```
Comment (ASLH68) [inline Assets/GameAssets/Player/PlayerScripts/PlayerInventory.cs:24]: private

File: Assets/GameAssets/World/WorldScripts/Interactables/Specific Interactables/SpannerInteractable.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+/// <summary>
+/// Provides the functionality for the player to pickup the spanner
+/// </summary>
+public class SpannerInteractable : MonoBehaviour, IPlayerInteractable
+{
+    /// <summary>
+    /// Implements the OnInteractedByPlayer function from IPlayerInteractable
+    /// Called when the player presses the interact key while looking at this object
+    /// </summary>
+    void IPlayerInteractable.OnInteractedByPlayer()
```
Comment (ASLH68) [inline Assets/GameAssets/World/WorldScripts/Interactables/Specific Interactables/SpannerInteractable.cs:20]: Why isn't this just a `public void OnInteractedByPlayer` ?

**Review by ASLH68 — APPROVED** [review]

**Pull request merged**